### DISCUSSION TITLE
Meaning of ,histories'?

### DISCUSSION THREAD

**Discussion by bsarey** [nrc-cnrc/EGSnrc#1180] [Questions]
Dear Community, 

as someone that also uses the Geant4 framework I'm a little bit confused about the term of particle histories in DOSXYZnrc.

So in Geant4 one history describes (for the case of a linac/x-ray tube) one electron and all the secondaries resulting from the interaction with the target material. 

When I want to calculate the dose in a phantom in DOSXYZnrc with a phase space file scored in the Geant4 framework, do I have to give the NCASE variable the same number as the #histories registered in the phase space file to use all the secondary photons in it? 

Thanks

**Comment by rtownson** [comment]
In the case of phase-space sources, `ncase` is the number of particles to use from the source (not the number of histories). We also define a history as a primary and all of its secondaries, but I believe the term is occasionally misused in the documentation. If you would like any further clarification you are welcome to ask!
